app/assets/stylesheets/cclow/_geography-page.scss

```diff
@@ -44,6 +44,12 @@
         margin-right: 1.5rem;
       }
     }
+
+    .highlight {
+      color: $red;
+      font-weight: bold;
```

Comment:
Not that it makes much difference but it is always best to use the bold font instead of stretching the regular font.

Reply:
Ok, I will fix it, thanks)